src/vs/workbench/parts/tasks/electron-browser/terminalTaskSystem.ts

```diff
@@ -884,7 +884,7 @@ export class TerminalTaskSystem implements ITaskSystem {
 				// (or, if the task has no group, a terminal used by a task without group).
 				for (const taskId of this.idleTaskTerminals.keys()) {
 					const idleTerminalId = this.idleTaskTerminals.get(taskId)!;
-					if (this.terminals[idleTerminalId].group === group) {
+					if (idleTerminalId && this.terminals[idleTerminalId].group === group) {
```

Comment:
I added this check on `idleTerminalId` here to work around https://github.com/Microsoft/vscode/issues/68113